src/stories/components/svg/select-chevron-down.tsx

```diff
@@ -5,14 +5,26 @@ interface SelectChevronDownProps {
   height?: number;
   fill?: string;
   style?: React.CSSProperties;
+  onClick?: () => void;
+  className?: string;
 }
 
-export const SelectChevronDown = ({ width = 10, height = 6, fill = '#25273D', ...props }: SelectChevronDownProps) => (
+export const SelectChevronDown: React.FC<SelectChevronDownProps> = ({
+  width = 10,
+  height = 6,
+  fill = '#25273D',
+  onClick,
+  className,
+  ...props
+}: SelectChevronDownProps) => (
   <svg
+    className={className}
+    onClick={onClick}
     width={`${width}px`}
     height={`${height}px`}
     viewBox="0 0 10 6"
     fill={fill}
+    cursor={onClick ? 'pointer' : 'none'}
```

Comment:
`cursor: none` means hide the cursor, should be `default` the expected behavior right?

Reply:
you are right, done